apiserver/facade/interface.go

```diff
@@ -58,6 +58,8 @@ type Context interface {
 	Resources() Resources
 
 	// XXX for a given model either State() or CAASState() will return nil
+	IsCAAS() bool
```

Comment:
I don't mind too much but do we want these, given that you can check for `nil` from `State()` and `CAASState()`?